### DISCUSSION TITLE
Does mutate ignore null value since 1.0.0

### DISCUSSION THREAD

**Discussion by jacklimwenjie** [vercel/swr#1439] [Q&A]
Just asking out of curiosity, my project has a bug that appeared to be magically "fixed" after upgrading the `swr` package to version 1.0.0. 

However, after a deeper inspection, I found the bug is still there. 
The bug is caused by mutating a cache with a `null` value, causing a downstream component that expects the cache value to be an object and access one of its properties to crash. 

On the `swr` package version 1.0.0, the mutation with a `null` value now keeps the previously cached value, so the downstream component no longer crashes.

Is this something intended?

**Comment by shuding** [comment]
That sounds like a regression, you should be able to mutate a resource to `null`. I'll change this discussion to an issue for further investigation.
